src/content/troubleshooting/_index.en.md

```diff
@@ -58,7 +58,7 @@ If you are able to connect to remote service by using ServiceIP or globalIp, but
 This is good time to familiarize yourself with [Service Discovery Architecture](../architecture/service-discovery/) if you haven't already.
 
 ##### Check ServiceExport for your Service
-For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export.
+For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export. If it exists, check if the status of ServiceExport is "Exported"
```

Comment:
Add that if it is not Exported, note the Status to know the reason why.

Reply:
Done.